webpack.config.js

```diff
@@ -90,7 +90,8 @@ module.exports = function makeConfig(grunt) {
                 {
                     test: /\.jsx?$/,
                     loader: 'eslint-loader',
-                    exclude: shouldExclude
+                    // superdesk apps handle their own linter
+                    exclude: (p) => shouldExclude(p) || sdConfig.apps.some(app => p.indexOf(app) > -1),
```

Comment:
The point is that we want to exclude `node_modules` from being linted, not that we want to exclude node_modules, but not apps, or apps, which is redundant. It's like saying: _"exclude the path if it's in node_modules, but it's not an app OR if it's an app"_ versus simply saying _"exclude node_modules"_

Your version just confuses things. "It works" is not what we are after. We are after "it's maintainable and readable".